CHANGELOG.md

```diff
@@ -90,6 +90,7 @@ the detailed section referring to by linking pull requests or issues.
 * Add public Api to DPF and make `HttpDataSource` more generic (#699)
 * Add an in-memory data plane store (#705)
 * Introduce extensions for synchronous data transfer using data plane (#711)
+* Add a deleteById method to the AssetLoader interface (#880)
```

Comment:
wrong line :) Please put at line 25, it is a as-yet unreleased feature.

Reply:
Oops. Good catch! :)